bears/general/FilenameBear.py

```diff
@@ -26,6 +27,8 @@ class FilenameBear(LocalBear):
                                   }
 
     def run(self, filename, file,
+            max_len_path_unix: int=4096,
+            max_len_path_windows: int=260,
```

Comment:
Just thinking, do you mean path length or filename length? Paths can be different on different environments.

Also:
Change name pattern to `max_path_length_...` instead of `max_len_path_...`

Reply:
Should I check only filename length or filename and path length?